### PR TITLE
Add reference link to enhanced navigation and form handling

### PR DESCRIPTION
**Pull request by Kissaki**
Link to reference documentation of what the feature that can be disabled actually even is.

Without the link, the reader has to already know or search and discover for themselves what the concept even is.

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/blazor/fundamentals/startup.md](https://github.com/dotnet/AspNetCore.Docs/blob/9d1421f65b9d86ebcd679b41d126879e76c4fcfd/aspnetcore/blazor/fundamentals/startup.md) | [ASP.NET Core Blazor startup](https://review.learn.microsoft.com/en-us/aspnet/core/blazor/fundamentals/startup?branch=pr-en-us-33470) |

### CODE REVIEW

File: aspnetcore/blazor/fundamentals/startup.md [hunk 1]
```diff
@@ -828,7 +828,7 @@ Standalone Blazor WebAssembly:
 
 *This section applies to Blazor Web Apps.*
 
-To disable enhanced navigation and form handling, set `disableDomPreservation` to `true` for `Blazor.start`.
+To disable [enhanced navigation and form handling](xref:blazor/fundamentals/environments/routing#enhanced-navigation-and-form-handling), set `disableDomPreservation` to `true` for `Blazor.start`.
```
Comment (guardrex) [inline aspnetcore/blazor/fundamentals/startup.md:831]:
```suggestion
To disable [enhanced navigation and form handling](xref:blazor/fundamentals/environments/routing#enhanced-navigation-and-form-handling), set `disableDomPreservation` to `true` for `Blazor.start`:
```

**Review by guardrex — APPROVED** [review]
# 👍

**Comment by guardrex** [comment]
Ooooops. I'll check and see what went wrong here.

File: aspnetcore/blazor/fundamentals/startup.md [hunk 2]
```diff
@@ -828,7 +828,7 @@ Standalone Blazor WebAssembly:
 
 *This section applies to Blazor Web Apps.*
 
-To disable enhanced navigation and form handling, set `disableDomPreservation` to `true` for `Blazor.start`.
+To disable [enhanced navigation and form handling](xref:blazor/fundamentals/environments/routing#enhanced-navigation-and-form-handling), set `disableDomPreservation` to `true` for `Blazor.start`:
```
[the remark this answers is not in the document]
Comment (Kissaki) [reply aspnetcore/blazor/fundamentals/startup.md]: oops, thanks for the fixup

**Pull request merged**